aiida_vasp/workchains/bands.py

```diff
@@ -227,3 +231,10 @@ def seekpath_structure_analysis(structure, parameters):
     """
     from aiida.tools import get_explicit_kpoints_path
     return get_explicit_kpoints_path(structure, **parameters.get_dict())
+
+
+@calcfunction
```

Comment:
This is perfect I think.

Reply:
@atztogo Also, can you add a test that checks that the labels are present. I suspect checking if they are correct is a much bigger task and more a task for `seekpath` so let us not do that.

Reply:
OK. I will do. Thanks for your suggestion.